docs/source/gs-processing/developer/input-configuration.rst

```diff
@@ -447,6 +447,15 @@ arguments.
         will be considered as an array. For Parquet files, if the input type is ArrayType(StringType()), then the
         separator is ignored; if it is StringType(), it will apply same logic as in CSV.
 
+-  ``huggingface``
+
+   -  Transforms a text feature column to tokens or embeddings with different Hugging Face models, enabling nuanced understanding and processing of natural language data.
+   -  ``kwargs``:
+
+      - ``normalizer`` (String, required): It should be "tokenize_hf".
+      - ``bert_model`` (String, required): It should be the identifier of a pre-trained model available in the Hugging Face Model Hub.
```

Comment:
Have we tested that this works with models downloaded from the internet for both EMR Serverless and SageMaker?

Reply:
For SageMaker, the internet access works well, nothing else needs to be deployed. For EMR Serverless, either NAT or local cache in the docker image is necessary. Either of these two options can work.

Reply:
Let's include the necessary changes to the 0.2.2 EMR-S Dockerfile in this PR then.

Reply:
I'm also a bit confused here, if we don't support embedding generation how is the HF model relevant, are there different tokenization models?

Reply:
Cause you need the pretrained models to do the tokenization: https://huggingface.co/transformers/v3.0.2/model_doc/auto.html#autotokenizer